src/client/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,52 @@
+.avatar-student-comp {
+  height: 316.9661865234375px;
+  width: 30%;
+  left: 14.5313720703125px;
+  top: 36.3284912109375px;
+  border-radius: 50%;
+}
+
+.avatar-mentor-comp {
+  height: 316.9661865234375px;
+  width: 30%;
+  left: 14.5313720703125px;
+  top: 36.3284912109375px;
+  border-radius: 50%;
+}
+
+.avatar-admin-comp {
+  height: 316.9661865234375px;
+  width: 30%;
+  left: 14.5313720703125px;
+  top: 36.3284912109375px;
+  border-radius: 50%;
+}
+
+.all-avatar-images {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+@media all and (max-width: 480px) {
+  .all-avatar-images {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
```

Comment:
this is the same, no need for repeating
when using media queries you should put there only CSS that is different for that particular screen size